crates/signer/src/signer_v2/mod.rs

```diff
@@ -186,13 +186,37 @@ impl TxSigner<Signature> for GenericSigner {
 #[serde(untagged)]
 #[non_exhaustive]
 pub enum SignerConfig {
-    /// Hexadecimal private key
+    /// ECDSA hexadecimal private key.
```

Comment:
All of the options are for [ECDSA](https://es.wikipedia.org/wiki/ECDSA) so that part can be misleading.

```suggestion
    /// Hex-encoded private key plaintext.
```